src/cmd.ts

```diff
@@ -134,14 +134,20 @@ function createCommandWithSharedOptions(name, description) {
 		)
 		.option(
 			"-e, --include-episodes",
-			"Include single-episode torrents in the search",
+			"Include all episode torrents in the search",
 			fallback(fileConfig.includeEpisodes, false)
 		)
+		.option(
+			"--include-single-episodes",
+			"Include only single episodes torrents in the search",
+			fallback(fileConfig.includeSingleEpisodes, false)
+		)
 		.option(
 			"--no-include-non-videos",
 			"Don't include torrents which contain non-videos"
 		)
 		.option("--no-include-episodes", "Don't include episode torrents")
+		.option("--no-include-single-episodes", "Don't include episodes from a season pack")
```

Comment:
```suggestion
		.option("--no-include-single-episodes", "Don't include single episodes")
```

Reply:
Remove this option due to redundancy with --no-include-episodes.